bigtable/integration_test.go

```diff
@@ -885,7 +918,7 @@ func TestIntegration_Read(t *testing.T) {
 	}{
 		{
 			desc: "read all, unfiltered",
-			rr:   RowRange{},
```

Comment:
I'm concerned that this had to change....it implies a breaking change. I think the root of the problem is that rangeUnbounded is not mapped to 0 in your enum

Reply:
True. I think changing the default rangeBound to rangeUnbounded will preserve the behavior of RowRange better - although direct use of the RowRange constructor is discouraged

Reply:
Agreed, but its almost definitely being used by endusers.